webapp/components/integrations/components/add_command/add_command.jsx

```diff
@@ -1,13 +1,11 @@
-// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Small note here, we don't typically update the copyright date after the file was created, but that's not a particularly strict rule